<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoBottomBar</class>
 <widget class="QWidget" name="VideoBottomBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>43</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="homeBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="menuBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>menu</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
